source/interprocedural_analyses/taint/taintConfiguration.ml

```diff
@@ -1517,6 +1545,49 @@ let validate ({ Heap.sources; sinks; transforms; features; _ } as configuration)
     |> Result.combine_errors_unit
     |> Result.map_error ~f:List.concat
   in
+  let compare_locations previous_location location =
+    match previous_location, location with
+    | Some previous_location, Some location ->
+        JsonAst.LocationWithPath.equal previous_location location
+    | None, None -> true
+    | _ -> false
+  in
+  let ensure_list_unique_with_location ~get_key ~get_error ~get_location elements =
+    let table = Hashtbl.create (module String) in
+    let seen_errors = Hash_set.create (module Error) in
+    let ensure_unique element =
+      let key = get_key element in
+      let location = get_location element in
+      match Hashtbl.find table key with
+      | None ->
+          let _ = Hashtbl.add table ~key ~data:location in
```

Comment:
nit: since `Hashtbl.add` returns `unit` it's better to use `()` instead of `_`. 

Using `let _ = ..` is a bit dangerous because you could forget to pass a parameter and it wouldn't complain, i.e `let _ = Hashtbl.add table` compiles.

```suggestion
          let () = Hashtbl.add table ~key ~data:location in
```

Reply:
Thanks for teaching me that!